lib/spacesuit/router.ex

```diff
@@ -40,7 +40,10 @@ defmodule Spacesuit.Router do
       |> process_headers
       |> add_all_actions
 
-    {route, Spacesuit.ProxyHandler, compiled_opts}
+    constraints = Map.get(compiled_opts, :constraints) || []
```

Comment:
I realized there is a form of `Map.get` that takes a default, like `constraints = Map.get(compiled_opts, :constraints, [])`